scenarios.md

```diff
@@ -230,6 +230,21 @@ A weakness is discovered in a widely used cryptographic algorithm and a decision
 1. Key revocation, chain of trust, etc. must all work for the expected lifetime of a version of the client software while these changes are made.
 1. The actions that different parties need to perform must be clearly articulated, along with the result of not performing those actions.
 
+### Scenario #10: Revoking a Signature
+
+A signer determines that a signed artifact is no longer trusted. This could be a result of accidentally signing the wrong artifact, or a CVE discovered in the image and patched in a newer release. It should be possible for the signer to revoke the trust in that vulnerable artifact.
+
+1. A CVE is discovered in a signed artifact.
+1. A new artifact is published with a fix for the CVE.
+1. The signature on the vulnerable artifact is revoked.
+1. Users pulling the old artifact that verify signatures will receive an error forcing them to upgrade to the newer artifact.
```

Comment:
The tooling should go as far as throwing an error. The user pulling the artifact should determine what the appropriate next step is.

Reply:
Thanks, I've reworded this one.